renovate.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:recommended"],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "automerge": true
+  },
+  "npm": {
+    "fileMatch": ["(^|/)(package\\.json)$"]
+  },
+  "ignoreDeps": [
+    "@nrwl/*",
+    "@nx/*",
+    "typescript",
+    "swc-loader",
+    "@swc/helpers"
+  ],
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["minor", "patch"],
```

Comment:
update if not a major version bump, at 2am and merge if everything passes, only devDependencies.